### PR TITLE
Add firmware version reporting for Develco EMIZB-132 and SMSZB-120, support for Climax KP-23EL-ZBS-ACE keypad

### PR DESCRIPTION
**Pull request by olemyhre**

### CODE REVIEW

File: converters/fromZigbee.js [hunk 1]
```diff
@@ -716,6 +716,23 @@ const converters = {
             };
         },
     },
+    ias_keypad: {
+        cluster: 'ssIasZone',
+        type: 'commandStatusChangeNotification',
+        convert: (model, msg, publish, options, meta) => {
+            const zoneStatus = msg.data.zonestatus;
+            return {
+                alarm_1: (zoneStatus & 1) > 0,
```
Comment (Koenkk) [inline converters/fromZigbee.js:725]: Are all attributes used by the keypad?

Comment (olemyhre) [reply converters/fromZigbee.js:725]: I can't find any good documentation on this device, but tamper and restore_reports are reported by the keypad, not sure about battery_low. The others are probably not used.

Comment (olemyhre) [reply converters/fromZigbee.js:725]: image

Comment (Koenkk) [reply converters/fromZigbee.js:725]: Please remove the ones not used by the device (with that I mean those which never change)

**Comment by olemyhre** [comment]
Done

**Comment by Koenkk** [comment]
Thanks!

**Pull request merged**